packages/harpparts/src/apparatus/utils/build-apparatus/index.ts

```diff
@@ -0,0 +1,21 @@
+import { reedArrayToMatrices } from '../reed-array-to-matrices'
+import type { Apparatus } from '../../types'
+import { ValvingIds } from '../../../valving'
+import type { TuningIds } from '../../../tuning'
+import { getTuning } from '../../../access-parts'
+
+export const buildApparatus = (
+  tuningId: TuningIds,
+  valvingId: ValvingIds = ValvingIds.NotValved
```

Comment:
Do we want to leave this as a non-mandatory field? I don't think so, and I don't think that the effort of updating all the references is a good reason not to.

Reply:
Done